docs/user.md

```diff
@@ -281,6 +281,23 @@ spec:
     s3_force_path_style: true
 ```
 
+## Setting up a standby cluster
+
+Standby clusters are like normal cluster but they are streaming from a remote cluster. As the first version of this feature, the only scenario covered by operator is to stream from a wal archive of the master. Following the more popular infrastructure of using Amazon's S3 buckets, it is mentioned  as s3_wal_path here. To make a cluster as standby add a section standby in the YAML file as follows.
+
+```yaml
+spec:
+  standby:
```

Comment:
there would be multiple modes of standby operation and every mode requires own set of configs. IMHO a bit of future proofing might help, such as:
```
standby:
  wal_shipping:
     s3_bucket: ....
```
then future streaming replication would look like:

```
standby:
  streaming:
     host: ...
     slot: ...
```